project.clj

```diff
@@ -62,7 +62,7 @@
                                   [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
                                   [org.clojure/tools.namespace "0.2.4"]
                                   [org.clojure/java.jmx "0.2.0"]
-                                  [spyscope "0.1.4"]
+                                  [spyscope "0.1.5" :exclusions [clj-time]]
```

Comment:
I bumped the spyscope dependency b/c my Intelli-J IDE was complaining about the conflict between the older clj-time dep that the previous spyscope version had vs. the one that other Clojure libraries needed.  This seemed like the cleanest way to overcome the problem.  Also had to bump the clojure dep to 1.6 because the newer spyscope depends upon it.